### ISSUE TITLE
Get runs with is completed returns Invalid characters in URI: [/api/v2/get_runs/12?is_completed]

### ISSUE THREAD

**Issue by Scot3004** [DamianOsipiuk/testrail-js-api#4]
# Actual behavior
When I try to get the runs of my TestRail instance by ussing the parameter {is_completed: 0}
with the function `testrailAPI.getRuns(projectId, { is_completed: 0 })`
Then the response payload is
```json
{
  "error": "Invalid characters in URI: [/api/v2/get_runs/12?is_completed]"
}
```

# Expected behavior
The test runs should be returned by the method getRuns

**Comment by DamianOsipiuk** [comment]
@Scot3004 Fixed in v0.1.6
The cause of the bug was that query string params were actually not the only one. API URL is actually the first parameter.
I broke that in the last version.

**Comment by Scot3004** [comment]
Impressively fast, thanks you for it, I'll test it for close
